core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2290,8 +2290,13 @@ private void loadMetadataInternal(RpcContext rpcContext, LockedInodePath inodePa
               LoadMetadataOptions loadMetadataOptions =
                   LoadMetadataOptions.defaults().setLoadDescendantType(DescendantType.NONE)
                       .setCreateAncestors(false).setUfsStatus(childStatus);
-              loadMetadataInternal(rpcContext, tempInodePath, loadMetadataOptions);
-
+              try {
+                loadMetadataInternal(rpcContext, tempInodePath, loadMetadataOptions);
+              } catch (Exception e) {
+                LOG.debug("Failed to loadMetadata: inodePath={}, options={}.", tempInodePath.getUri(),
```

Comment:
can we make this `info`-level? We use `debug` below, but that's because we also propagate the exception. We don't want metadata loads to silently fail here

Reply:
done